python/lsst/meas/base/base.py

```diff
@@ -129,29 +154,29 @@ class ForcedTransformedCentroidPlugin(ForcedPlugin):
     @endcode
     """
     def decorate(PluginClass):
-        PluginClass.registry.register(name, PluginClass)
+        PluginClass.registry.register(name, PluginClass, canApCorr=canApCorr)
         return PluginClass
     return decorate
 
 
 class PluginMap(collections.OrderedDict):
     """!
-    Map of plugins to be run for a task
+    Map of plugins (instances of subclasses of BasePlugin) to be run for a task
 
-    We assume Plugins are added to the PluginMap according to their "Execution Order", so this
+    We assume plugins are added to the PluginMap according to their "Execution Order", so this
     class doesn't actually do any of the sorting (though it does have to maintain that order,
     which it does by inheriting from OrderedDict).
     """
 
     def iter(self):
-        """Call each plugin in the map which has a measure() method
+        """Call each plugin in the map which has a .config.measure() method
         """
         for plugin in self.itervalues():
             if plugin.config.doMeasure:
                 yield plugin
 
     def iterN(self):
-        """Call each plugin in the map which has a measureN() method
+        """Call each plugin in the map which has a .config.measureN() method
```

Comment:
How do you enable measure() or measureN in the config? I think it would be worth saying in the doc string. Right now I have this, which seems to beg the question:
{code}
        """Call each plugin in the map which has a measureN() method and is enabled in the config
        """
{code}